src/figuresClasses.ts

```diff
@@ -1,19 +1,84 @@
 export interface Figure {
+  shape: 'triangle'| 'circle' | 'rectangle',
+  color: 'red' | 'green' | 'blue',
+  getArea(): number,
+}
 
+function getTruncNum(number: number): number {
+  return Math.trunc(number * 100) / 100;
 }
 
 export class Triangle {
+  public shape: Figure['shape'] = 'triangle';
+
+  constructor(
+    public color: Figure,
```

Comment:
It should be the `Figure['color']` as I see

Reply:
Which type for color do you have?
The same as you use type for shape.
'Figure' is the whole Interface but you need only color here.